README.md

```diff
@@ -52,15 +52,15 @@ This will return a list of dictionaries looking somewhat like this:
 You can also add the `languages` param if you want to make sure the transcripts are retrieved in your desired language (it defaults to english).
 
 ```python
-YouTubeTranscriptApi.get_transcripts(video_ids, languages=['de', 'en'])
+YouTubeTranscriptApi.get_transcripts("video_ids", languages=['de', 'en'])

```

Comment:
This would definitely be misleading, as `get_transcripts` must receive a list and not a string. Therefore, you'd have to do `['video_id1', 'video_id2']` instead.